packages/toast/src/toast.class.tsx

```diff
@@ -49,15 +49,19 @@ class Toaster {
   }
 
   notify = (message: ToastMessage, options: Partial<ToastOptions> = {}) => {
-    return this.createToast?.(message, options)
+    return this.createToast ? this.createToast(message, options) : undefined
```

Comment:
refactored optional chaining in favor of if/ternaries for a small bundlesize gain

Reply:
Is it worth that refactor optional chaining ?It keeps code clean. Maybe the ts compiler will imporve the bundlesize in the feature?

Reply:
while writing this, turns out I measured the wrong data and forget the `options = {}` assignment when comparing... its not a reduction of 28% (which I'd say is a free, although micro, optimization) but only 3.4% :/ so I'll likely revert this

thanks!

Reply:
gone :]